es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 **1. Machine Learning tips and tricks cheatsheet**
 
-&#10230;
+&#10230; Hoja de trucos y ayudas de aprendizaje automático
 
 <br>
 
 **2. Classification metrics**
 
-&#10230;
+&#10230; Métricas de clasificación
 
 <br>
 
 **3. In a context of a binary classification, here are the main metrics that are important to track in order to assess the performance of the model.**
 
-&#10230;
+&#10230; En el contexto de una clasificacion binaría, estas son las principales métricas que se tienen que seguir para evaluar el desempeño del modelo
 
 <br>
 
 **4. Confusion matrix ― The confusion matrix is used to have a more complete picture when assessing the performance of a model. It is defined as follows:**
 
-&#10230;
+&#10230; Matriz de confusión ― La matriz de confusión es usada para tener un mejor panorama cuandpo se evalua el desempeño del modelo. Se define como:
 
 <br>
 
 **5. [Predicted class, Actual class]**
 
-&#10230;
+&#10230; [Clase predecida, Clase Real]
 
 <br>
 
 **6. Main metrics ― The following metrics are commonly used to assess the performance of classification models:**
 
-&#10230;
-
+&#10230; Métricas principales ― Las siguientes métricas son comumente usadas para evaluar el desempeño de modelos clasificatorios
+ 
 <br>
 
 **7. [Metric, Formula, Interpretation]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Interpretación]*
 
 <br>
 
 **8. Overall performance of model**
 
-&#10230;
+&#10230; Desempeño en general del modelo
 
 <br>
 
 **9. How accurate the positive predictions are**
 
-&#10230;
+&#10230; ¿Qué tan precisan son las precisiones positivas?
 
 <br>
 
 **10. Coverage of actual positive sample**
 
-&#10230;
+&#10230; Covertura de las pruebas positivas
 
 <br>
 
 **11. Coverage of actual negative sample**
 
-&#10230;
+&#10230; Covertura de las pruebas negativas
 
 <br>
 
 **12. Hybrid metric useful for unbalanced classes**
 
-&#10230;
+&#10230; Métrica híbrida útil para clases no balanceadas
 
 <br>
 
 **13. ROC ― The receiver operating curve, also noted ROC, is the plot of TPR versus FPR by varying the threshold. These metrics are are summed up in the table below:**
 
-&#10230;
+&#10230; Curva Curva de operación del receptor (ROC) ― La curva receptora de operaciones, por sus siglás en inglés ROC, es la gráfica de los Tasa de verdaderos positivos" (TPR) contra los Tasa de falsos positivos (FPR) variando el umbral. Estas métricas son resumidas en la tabla siguiente: 
```

Comment:
* You repeat _Curva_ at the beginning of the sentence.
* When you are going to introduce an English acronym the correct way to do it is like:

> ... bla bla bla (ROC, por sus siglás en Inglés).

* Remove the extra " just after _Tasa de verdaderos positivos_.